Formula/l/lv.rb

```diff
@@ -33,6 +32,7 @@ class Lv < Formula
 
   # See https://github.com/Homebrew/homebrew-core/pull/53085.
   # Further issues regarding missing headers reported upstream to nrt@ff.iij4u.or.jp
+  # TODO: send MR to https://salsa.debian.org/debian/lv
```

Comment:
( I can't do this right now due to an issue of their gitlab infra: https://salsa.debian.org/salsa/support/-/issues/424 )

Reply:
Submitted https://salsa.debian.org/debian/lv/-/merge_requests/3